src/app/proposals/choose/page.tsx

```diff
@@ -16,7 +16,10 @@ export default function ChooseProposal() {
   // Closes the info panel when a click occurs outside the specified container.
   const cardsRef = useRef<HTMLDivElement>(null)
   const infoPanelRef = useRef<HTMLDivElement>(null)
-  useClickOutside([cardsRef, infoPanelRef], () => setChosenProposal(null))
+  useClickOutside(
+    [cardsRef, infoPanelRef].filter((ref): ref is RefObject<HTMLDivElement> => ref.current !== null),
```

Comment:
[NIT] just to be on the safe side

```suggestion
    [cardsRef, infoPanelRef].filter((ref): ref is RefObject<HTMLDivElement> => !!ref.current),
```